src/test/ui/terr-sorts.stderr

```diff
@@ -2,7 +2,10 @@ error[E0308]: mismatched types
   --> $DIR/terr-sorts.rs:10:14
    |
 LL |     want_foo(b);
-   |              ^ expected struct `Foo`, found struct `Box`
+   |              ^
+   |              |
+   |              expected struct `Foo`, found struct `Box`
+   |              help: try dereferencing the `Box`
```

Comment:
Should I use `span_suggestion_verbose` instead to improve cases like this?

Reply:
I think so, r=me after that.